build/docker/dev/docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
       - ./../../..:/app
     environment:
       APP_ENV: dev-docker
+      TRINO-GATEWAY_AUTH_TOAUTHENTICATE: true
```

Comment:
nit: replace with `TRINO_GATEWAY_AUTH_AUTHENTICATE`

Reply:
`TRINO-GATEWAY` is code generated prefix for all variables.

Replacing to `TRINO-GATEWAY_AUTH_AUTHENTICATE`